cypress/support/helpers/athenaReportHelper.js

```diff
@@ -30,56 +34,49 @@ export function verifyCountersInUI(counters){
     cy.get(learnersReportPageElements.LBL_TEACHERS_COUNT).should('have.text', counters.teachersCount);
 }
 
-/**
- * This api is to build the athena report url with filters from UI.
- * filterValues is a list of objects [ {name: 'filter name', id: 'filter id' } ] format.
- * filter name is used in UI to filter.
- * filter id is used in API to query.
- */
-export function addLearnerReportFilter(learnerUrl, filterKey, filterValues){
-    if(filterValues === null || !filterValues ){
-        return learnerUrl;
+export function getLearnerReportUrl(props){
+    let learnerReportUrl = 'https://learn-report.staging.concord.org' + props.queryUrl;
```

Comment:
the learner-report.staging.concord.org should be a variable since it will be different depending on which portal host you are testing against.